t.py

```diff
@@ -0,0 +1,86 @@
+#coding=utf-8
+
+from flask import Flask
+import requests
+import json
+import sys
+import time
+import getpass
+
+USERNAME = "" 
+PASSWORD = ""
+
+s = "" # session
+
+URL = "http://kuas.grd.idv.tw:14768/" #API URL
+
+def setData():
+    global USERNAME,PASSWORD
+    USERNAME = raw_input("Enter Your Account:")
+    PASSWORD = getpass.getpass("Enter Your Password:")
+
+    k = list()
+    ltime = time.localtime(time.time())
```

Comment:
Please use descriptive variable name, `local_time` is better than `ltime`, for future you and others developers.